lib/content-services/src/lib/content-node-selector/content-node-selector.component.spec.ts

```diff
@@ -393,4 +393,32 @@ describe('ContentNodeSelectorComponent', () => {
             expect(tabGroup).toBe(undefined);
         });
     });
+
+    describe('Breadcrumb', () => {
+        let contentNodePanel;
+
+        beforeEach(() => {
+            contentNodePanel = fixture.debugElement.query(By.directive(ContentNodeSelectorPanelComponent));
+            spyOn(component, 'onBreadcrumbFolderTitleEvent');
+        });
+
+        it('should emit a breadcrumbFolderTitleEvent when the folder is changed', () => {
```

Comment:
The event emitter is part of the content-node-selector-panel component and the unit tests should be part of that component and not part of the content-node-selector component. In the panel component you can spy on the event emitter to check what is emitted